test/fixtures/test/states.json

```diff
@@ -46,18 +35,29 @@
   {
     "id": "77777",
     "name": "THMG Transfered",
-    "sequence": 4,
+    "sequence": 400,
     "status": "done",
     "options": {
       "iconType": "fas",
       "icon": "arrow-right-arrow-left",
       "color": "purple"
     }
   },
+  {
+    "id": "55555",
+    "name": "Done",
+    "sequence": 300,
+    "status": "done",
+    "options": {
+      "iconType": "fas",
+      "icon": "circle-check",
+      "color": "green"
+    }
+  },
```

Comment:
This confirms that the `comparator: 'sequence'` works. With the order of these changed, removing the `comparator: 'sequence'` causes tests to fail.

But I’m worried that this file might be changed in the future and we’ll unknowingly no longer be testing the comparator.

So maybe we want to add a test to the filters sidebar where we do a `routeStates(fx => {})` and we test this specifically without modifying the `fixtures/test/states.json` file?

Reply:
Well, maybe that won't work 🤔 . Since any changes in the `routeStates(fx => {})` would be applied after the data is already received from the api and therefore after the `comparator` is applied?

Reply:
`routeStates` would modify the order if place before the `.visit`. It's probably worth testing in case `66666` and `77777` are removed

Reply:
I agree, I just added a test for that.